README.md

```diff
@@ -34,6 +34,8 @@ Images and real-life data make text more engaging and powerful, but be sure what
 you use is appropriately licensed and available. Here again, even a rough idea
 for artwork can be polished by others.
 
+Pull requests are reviewed in markdown format. The 
+
 ### Why Jupyter Notebooks?
```

Comment:
@melissawm , I didn't want to delete existing text without a conversation. 

__Why ~~Jupyter~~ MyST-NB Notebooks?__

This would be a good place to discuss why we are using MyST-NB. Once you have Jupyter+Jupytext, the MyST markdown can be a direct replacement for ipynb without outputs. Since we do not review code outputs, it makes sense to avoid .json files that are meant to couple code+output.

Reply:
I see. But I still think most users will come from a plain notebook experience, and might not know what MyST NB is. My thinking was that at some point we could add an action to let the user upload a .ipynb file and the action would create the md file using jupytext - the user wouldn't even need to do that if they didn't want to. 

Maybe @rossbar can also chip in here?

Reply:
That's true. That was the same thinking I had about discussing common mark vs myst-markdown in this Readme. 

I think I should take my own advice and keep this doc simple. 

I do think it would help to discuss these topics in another document. @bjnath's tutorial makes sense to me.

Reply:
I agree with keeping it simple - there's a lot of terms and crosstalk (e.g. MyST the markdown flavor, the `myst-parser` which parses MyST-flavored markdown, `MyST-NB` which supports **both** a text-based notebook format *and* MyST-flavored markdown, etc.)

Fortunately none of this stuff should really be relevant for users. My vote would be to minimize trying to disentangle the above mess in our docs and state that `numpy-tutorials` uses a markdown-based format for Jupyter notebooks. Setting up a gh-action to auto-convert ipynb-to-md on push would be straightforward, though it might be more user-friendly to simply do so (with any necessary discussion) as part of the review process. There's a lot of flexibility here so I'm sure we can find an acceptable solution w/ more feedback from authors & reviewers.

Reply:
> Fortunately none of this stuff should really be relevant for users. My vote would be to minimize trying to disentangle the above mess in our docs and state that `numpy-tutorials` uses a markdown-based format for Jupyter notebooks. Setting up a gh-action to auto-convert ipynb-to-md on push would be straightforward, though it might be more user-friendly to simply do so (with any necessary discussion) as part of the review process. 

Yes, it would definitely help to leave out the "disentangling" discussion, especially in this front-facing README. 

I would prefer the notebook pairing process for users (i.e. they have both `.ipynb` and `.md` locally, but submit the `.md`). If they use the jupytext extension, then they don't need a cli approach. If they prefer cli, then the option is still there. In both cases, users can see what is being reviewed and they can update the local ipynb as the review occurs.